packages/core/test/invoke.test.ts

```diff
@@ -836,6 +836,50 @@ describe('invoke', () => {
         })
         .start();
     });
+
+    it('should not reinvoke root-level invocations', (done) => {
```

Comment:
a similar test could be written for entry actions - they are handled almost the same internally, but externally they look very different

it would be also great if the test title would include some more detail about this specific test because as it is it's a little bit vague